velox/exec/GroupingSet.cpp

```diff
@@ -277,13 +277,14 @@ void GroupingSet::initializeGlobalAggregation() {
 
   // Row layout is:
   //  - null flags - one bit per aggregate,
-  // - uint32_t row size,
+  //  - uint32_t row size,
   //  - fixed-width accumulators - one per aggregate
   //
   // Here we always make space for a row size since we only have one
   // row and no RowContainer.
   int32_t rowSizeOffset = bits::nbytes(aggregates_.size());
-  int32_t offset = rowSizeOffset + sizeof(int32_t);
+  // Fixed-width accumulators must be word aligned.
+  int32_t offset = bits::roundUp(rowSizeOffset + sizeof(int32_t), 16);
```

Comment:
Without this, the LongDecimal accumulator is segfaulting. Hence rounding off to 2 words.